web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/LeggTilFormueRevurderingRoute.kt

```diff
@@ -0,0 +1,109 @@
+package no.nav.su.se.bakover.web.routes.revurdering
+
+import arrow.core.Either
+import arrow.core.NonEmptyList
+import arrow.core.getOrHandle
+import arrow.core.left
+import arrow.core.right
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.su.se.bakover.common.serialize
+import no.nav.su.se.bakover.domain.Brukerrolle
+import no.nav.su.se.bakover.service.revurdering.KunneIkkeLeggeTilFormuegrunnlag
+import no.nav.su.se.bakover.service.revurdering.LeggTilFormuegrunnlagRequest
+import no.nav.su.se.bakover.service.revurdering.RevurderingService
+import no.nav.su.se.bakover.web.Resultat
+import no.nav.su.se.bakover.web.features.authorize
+import no.nav.su.se.bakover.web.routes.grunnlag.FormuegrunnlagJson
+import no.nav.su.se.bakover.web.routes.revurdering.FormueBody.Companion.toServiceRequest
+import no.nav.su.se.bakover.web.routes.søknadsbehandling.beregning.PeriodeJson
+import no.nav.su.se.bakover.web.sikkerlogg
+import no.nav.su.se.bakover.web.svar
+import no.nav.su.se.bakover.web.withBody
+import no.nav.su.se.bakover.web.withRevurderingId
+import no.nav.su.se.bakover.web.withSakId
+import java.util.UUID
+
+internal fun FormuegrunnlagJson.VerdierJson.depositumErMindreEllerLikInnskudd() = this.depositumskonto <= this.innskudd
+
+private data class FormueBody(
+    val periode: PeriodeJson,
+    val epsFormue: FormuegrunnlagJson.VerdierJson?,
+    val søkersFormue: FormuegrunnlagJson.VerdierJson,
+    val begrunnelse: String?,
+) {
+    private fun VerdierErGyldig() =
+        this.søkersFormue.depositumErMindreEllerLikInnskudd() && (
```

Comment:
Har vi noen "felles forståelse" for hvordan dette skal fungere/brukes? Kommer det tydelig fram i frontenden at det er en sammenheng mellom disse feltene? (nytt for meg)

Reply:
Jepp, det var en del diskusjon da vi holdt på med søknadsbiten.

Reply:
Men jeg tenkte meg at denne kunne ligget inne i tryCreaten til domeneklassen og ikke i routen :)